cmd/cstor-pool-mgmt/controller/pool-controller/handler.go

```diff
@@ -161,7 +169,44 @@ func (c *CStorPoolController) cStorPoolEventHandler(operation common.QueueOperat
 	return string(apis.CStorPoolStatusInvalid), nil
 }
 
-// cStorPoolAddEvent does import of pool, and, if it fails, attemps to create pool based on CSP CR state
+// cStorPoolCreate does create of pool, and, if it fails, returns CreateFailed
+func (c *CStorPoolController) cStorPoolCreate(cStorPoolGot *apis.CStorPool) (string, error) {
+	if !IsCStorPoolCreateStatuses(cStorPoolGot) {
```

Comment:
What is the intention of putting this check ? Can this function be called for other statuses that are not meant for create ?

Reply:
it can be.. but, as of now, it won't